docs/staking_claim_rewards.md

```diff
@@ -0,0 +1,20 @@
+---
+id: staking_claim_rewards
+title: Claim your Staking Rewards
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+At the end of every era, validator pools are assigned their [staking rewards](/staking_rewards) which consist of base rewards, era points and tips. These rewards, however, are not distributed automatically to the accounts of the validator and its nominators. This will only happen after the staking rewards have been claimed by **triggering a payout**. Staking rewards must be claimed **within 84 eras** after they have been earned. Once this period lapses, the relevant rewards information is deleted from the chain and the validator pool can no longer receive its rewards for that era. 
```

Comment:
```suggestion
At the end of every era, validator pools are assigned their [staking rewards](/staking_rewards) which consist of base rewards, era points rewards and tips. These rewards, however, are not distributed automatically to the accounts of the validator and its nominators. This will only happen after the staking rewards have been claimed by **triggering a payout**. Staking rewards must be claimed **within 84 eras** after they have been earned. Once this period lapses, the relevant rewards information is deleted from the chain and the validator pool can no longer receive its rewards for that era. 
```